.github/workflows/release.yml

```diff
@@ -640,6 +628,11 @@ jobs:
       AWS_S3_STATIC_BUCKET: ${{ vars.AWS_S3_STATIC_BUCKET }}
       AWS_REGION: ${{ vars.AWS_REGION }}
     steps:
+    - name: Finish early
+      if: ${{ inputs.auto }}
+      run: |
+        echo "skipping version info update for nightly patch"
+        exit 0
```

Comment:
we want this job to always run, so we can make sure that the tagging job always runs after it, if at all